v-next/hardhat/templates/02-mocha-ethers/scripts/deploy-rocket-from-script.ts

```diff
@@ -0,0 +1,23 @@
+import apolloModule from "../../../../example-project/ignition/modules/Apollo.js";
+import { createHardhatRuntimeEnvironment } from "@ignored/hardhat-vnext/hre";
+import hardhatIgnitionEthersPlugin from "@ignored/hardhat-vnext-ignition-ethers";
+
+const hre = await createHardhatRuntimeEnvironment({
+  plugins: [hardhatIgnitionEthersPlugin],
+});
+
+const { ignition } = await hre.network.connect();
+
+const { apollo } = await ignition.deploy(apolloModule);
```

Comment:
This is a Ethers `Contract` instance, as we do not have Typechain support.

Reply:
This is also true for hh2, right?

Reply:
This is also the case for HH2. We talked about adding it but never did. I think we should add it as part of the public alpha rollout, but we can decide that later.